tests/unit/api/test_simple.py

```diff
@@ -87,6 +88,16 @@ def test_selects(self, header, expected):
 
 
 class TestSimpleIndex:
+
+    @pytest.fixture
+    def db_request(self, db_request):
+        """Override db_request to add the Release Verification service"""
+        db_request.find_service = lambda svc, name=None, context=None: {
+            IIntegrityService: pretend.stub(),
+        }.get(svc)
+
+        return db_request
```

Comment:
Flagging: I think we want to drop this stub and below. This is probably what masked the underlying lack of a configured service.

Reply:
Removed in https://github.com/pypi/warehouse/pull/16546/commits/b151b821271882715432ec2922b230db55fa542b.